x/gov/client/cli/tx.go

```diff
@@ -49,6 +50,200 @@ var ProposalFlags = []string{
 	FlagDeposit,
 }
 
+// NewTxCmd returns the transaction commands for this module
+// governance ModuleClient is slightly different from other ModuleClients in that
+// it contains a slice of "proposal" child commands. These commands are respective
+// to proposal type handlers that are implemented in other modules but are mounted
+// under the governance CLI (eg. parameter change proposals).
+func NewTxCmd(cdc codec.Marshaler, txg tx.Generator, ar tx.AccountRetriever, msgSubmitProposalCtr func() types.MsgSubmitProposalI, pcmds []*cobra.Command) *cobra.Command {
```

Comment:
nit: I would shorten the name of the `msgSubmitProposalCtr` variable. I'd also break this function signature into multiple lines.

Reply:
Is just `ctr` fine?

Reply:
how about `newMsgFn func() types.MsgSubmitProposalI`

Reply:
Sure, will update